filters/gender_bias/README.md

```diff
@@ -0,0 +1,132 @@
+## gender bias filter
+
+## What type of a filter is this?
+
+This is a quadrilingual filter (for English, French, Polish and Russian languages), which filters a text corpus to measure gender fairness in regards of a female gender representation.
```

Comment:
Replace "in regards of a"  with "with respect to"

Reply:
Updated